addons/website_sale/views/sale_order_views.xml

```diff
@@ -60,6 +60,11 @@
                     class="btn-primary"
                     attrs="{'invisible': ['|', ('is_abandoned_cart', '=', False), ('cart_recovery_email_sent', '=', True)]}"/>
             </xpath>
+            <xpath expr="//button[@name='action_quotation_send' and @states='sent,sale']" position="after">
+                 <!-- The '| and the '|' opertors are necessary because draft state of the parent concatenate the domain -->
+                <button name="action_quotation_send" string="Send by Email" type="object" states="draft"
+                    attrs="{'invisible': ['|', '|', ('is_abandoned_cart', '=', False), ('cart_recovery_email_sent', '=', True)]}"/>
```

Comment:
I'd rather have a valid domain and explicitely add that we hide it outside of draft state instead of using `states` shortcut
```suggestion
                 <!-- The '| and the '|' opertors are necessary because draft state of the parent concatenate the domain -->
                <button name="action_quotation_send" string="Send by Email" type="object" states="draft"
                    attrs="{'invisible': ['|', '|', ('is_abandoned_cart', '=', False), ('cart_recovery_email_sent', '=', True), ('state', '!=', 'draft')]}"/>
```